### DISCUSSION TITLE
Should we release 0.69.7?

### DISCUSSION THREAD

**Discussion by Titozzz** [reactwg/react-native-releases#35] [Patches]
Released, please refer to https://github.com/reactwg/react-native-releases/discussions/47

---

Current Release: [0.69.6](https://github.com/facebook/react-native/releases/tag/v0.69.6)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1.

#### Local commits to backport to main

1.

**Comment by jayantdash** [comment]
@Titozzz We already have 0.70.1 previously. This 0.69.6 release is confusing.

**Reply by brunoducattii** [reply]
agree

**Reply by RalissonMattias** [reply]
I also find it confusing

**Comment by RicardoGuiliani** [comment]
@Titozzz from 0.70.1 to 0.69.6 and package.json commit says: "Bump react-native-gradle-plugin to 0.71.2" ??

**Reply by kelset** [reply]
you are not supposed to move from a newer minor (0.70) to an older one (0.69)

**Comment by putuoka** [comment]
Still can't auto upgrade using `npx react-native upgrade`

**Comment by retyui** [comment]
a small one but helpful: https://github.com/facebook/react-native/commit/475310dbbaec8048411edefc6cdddab330df7966

**Comment by cortinico** [comment]
Here the PR to include `exclusiveContent` on template for 0.69:
- https://github.com/facebook/react-native/pull/35217
